Firmware/mmu2_protocol_logic.h

```diff
@@ -330,31 +362,27 @@ class ProtocolLogic {
     static constexpr uint8_t regs8Count = 3;
     static_assert(regs8Count > 0); // code is not ready for empty lists of registers
     static const uint8_t regs8Addrs[regs8Count] PROGMEM;
-    uint8_t regs8[regs8Count];
+    uint8_t regs8[regs8Count] = { 0, 0, 0 };
```

Comment:
is `regs8` not zero initialised by the compiler? Adding {0, 0, 0} increases code size by 12 bytes.

Same applies to `regs16`

Reply:
:thinking: it has been like this before ... just the initialization was written in the constructor.
And I'd sacrifice those 12B for the sake of safety just in case this class gets used somewhere else in multiple instances.